exercises/practice/sgf-parsing/.meta/config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "authors": [],
```

Comment:
```suggestion
  "authors": ["tlphat"],
```

Since you ported the exercise, you deserve to be the author of it!

Reply:
Thanks, I've added myself as the author.